devtools/build_requires.rst

```diff
@@ -262,7 +262,7 @@ Making tool_requires affect the consumers package-ID
     This subsection should be considered a workaround, not a feature, and it might have other side effects, that will not be fixed
     as this is not recommended production code.
 
-As discussed above, the ``tool_requires`` do not affect at all the package ID. As they will not be present at all when the ``package_id``
+As discussed above, the ``tool_requires`` does not affect at all the package ID. As they will not be present at all when the ``package_id``
```

Comment:
```suggestion
As discussed above, the ``tool_requires`` do not affect at all the package ID. As they will not be present at all when the ``package_id``
```

I think this is good, it means plural, there can be several ``tool_requires``, no one of them affects package_id. The plural is also implicit in later ``As they will not be present...``

Reply:
You're right, sorry! Nice catch